environment_setup/setup_software.sh

```diff
@@ -105,11 +102,11 @@ echo "Installing Bazel"
 echo "================================================================"
 
 # Adapted from https://docs.bazel.build/versions/master/install-ubuntu.html#install-on-ubuntu
-sudo apt install curl gnupg
+sudo apt-get install curl gnupg
 curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
 echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
-sudo apt update
-if ! sudo apt install bazel -y ; then
+sudo apt-get update
+if ! sudo apt-get install bazel -y ; then
     echo "##############################################################"
```

Comment:
not blocking but curious why apt->apt-get?

Reply:
I saw some warnings in CI about apt interface not being stable. Link with more info: https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning